src/utility/control_sil/launch/control_sil_test.launch

```diff
@@ -0,0 +1,16 @@
+<!-- Launch file for the control software in the loop testing node -->
+
+
+<launch>
+  <!-- type-> executable , pkg->package, name-> node name-->
+  <node type="control_sil_sim" pkg="control_sil" name="control_sil_sim" output="screen" >
```

Comment:
Please include units for each of these parameters.

Reply:
Resolved in newest commit.